vendor/manifest

```diff
@@ -888,6 +888,14 @@
 			"branch": "v1"
 		},
 		{
+			"importpath": "gopkg.in/urfave/cli.v1",
+			"repository": "https://gopkg.in/urfave/cli.v1",
```

Comment:
Not directly related, just to improve the capability to print out NPE. I can separate it.